db/new_seeds/base/add_npq_registrations.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 npq_lead_providers = NPQLeadProvider.all
+cohort_2022 = Cohort.find_by(start_year: 2022)
```

Comment:
maybe we can loop over current and next cohort (to avoid hardcoding years) and create the same npq applications for both?